### PR TITLE
Fix READMEs

### PR DESCRIPTION
**Pull request by vraja2**
@optimizely/fullstack-devs @mauerbac

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +50,7 @@ config common for all modules.
 
 1. Clone the repo
   * `git clone git@github.com:optimizely/optimizely-ab-android-sdk.git`
```
Comment (elliotykim) [inline README.md]: This is out of date:
git@github.com:optimizely/android-sdk.git

Comment (vraja2) [inline README.md]: good catch

File: README.md [hunk 2]
```diff
@@ -12,11 +12,9 @@ Devel<br/>
 [![Apache 2.0](https://img.shields.io/github/license/nebula-plugins/gradle-extra-configurations-plugin.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 ## Overview
 
-Our Optimizely X Android solution allows you to easily run experiments anywhere in an Android application, even Services. The solution includes easy-to-use SDKs for experimenting in your code and tracking conversion events in Optimizely.
+This repository houses the Android SDK for Optimizely's Full Stack product. To find out more check out the [documentation](https://developers.optimizely.com/x/solutions/sdks/introduction/index.html?language=android&platform=mobile).
```
Comment (mauerbac) [inline README.md]: Optimizely's Mobile product

**Comment by mauerbac** [comment]
LGTM

**Pull request merged**